packages/twenty-server/src/engine/core-modules/auth/token/token.module.ts

```diff
@@ -15,13 +15,15 @@ import { WorkspaceSSOModule } from 'src/engine/core-modules/sso/sso.module';
 import { User } from 'src/engine/core-modules/user/user.entity';
 import { Workspace } from 'src/engine/core-modules/workspace/workspace.entity';
 import { DataSourceModule } from 'src/engine/metadata-modules/data-source/data-source.module';
+import { WorkspaceModule } from 'src/engine/core-modules/workspace/workspace.module';
 
 @Module({
   imports: [
     JwtModule,
     TypeOrmModule.forFeature([User, AppToken, Workspace], 'core'),
     TypeORMModule,
     DataSourceModule,
+    WorkspaceModule,
```

Comment:
token module is low level module, feels wrong to take depency on workspaceModule, this will likely result in circular depdency